zulipterminal/cli/run.py

```diff
@@ -312,6 +312,12 @@ def main(options: Optional[List[str]]=None) -> None:
                 sys.exit(1)
             boolean_settings[setting] = (zterm[setting][0] == valid_values[0])
 
+        # For threshold settings
+        int_settings = dict()
+        footlinks_threshold = zterm['footlinks_threshold'][0]
+        if footlinks_threshold.isnumeric():
+            int_settings['footlinks_threshold'] = int(footlinks_threshold)
+
```

Comment:
See review.

Reply:
So I should remove `footlinks=enabled` setting and keep only `footlinks_threshold` and also should change all `bool` settings to `int`?

Reply:
Yes, that's what I was suggesting - remove `footlinks` and only have `footlinks_threshold`.

I believe the current set of boolean settings are user-friendly with `enabled` and `disabled`. We shouldn't change all the settings to `int`.

Reply:
- I meant storing them in `Dict()` as `int`.